entrypoint.sh

```diff
@@ -54,7 +54,7 @@ git config user.email $ACTION_MAIL
 git pull https://${GH_PAT}@github.com/$OWNER/$REPO_NAME.wiki.git
 cd ..
 
-for i in $(find $MD_FOLDER -maxdepth 1 -type f -name '*.md' -execdir basename '{}' ';'); do
+for i in "$(find $MD_FOLDER -maxdepth 1 -type f -name '*.md' -execdir basename '{}' ';')"; do
```

Comment:
If MD_FOLDER have space you will end up with an error here.

Reply:
I tested... adding the `"` it was the only way to let it work correctly. Which kind of error are you thinking about @GregoireW ?

Without the `"` around the command, the for will split even over spaces, creating problems due to missing files.